### ISSUE TITLE
Prompt the user to confirm publishing a post without a title

### ISSUE THREAD

**Issue by fluiddot** [wordpress-mobile/WordPress-Android#17462]
> [!NOTE] 
Update: This issue needs further design review.
Internal discussion: pcdRpT-3gS-p2

Posts can be published without a title, however, in most cases, this might be performed by the user by mistake. Not having a title has downsides, especially related to SEO, as the title is used in several parts of the post like to form the Permalink and social meta tags.

An enhancement to incorporate in the publishing flow is to display a confirmation dialog when the user tries to publish a post without a title. The description of the dialog will inform the user about the benefits of setting a title, and will display two options:
- **Set title:** This option will stop the publishing flow and will focus on the title's text input.
- **Skip:** This option will continue with the publishing flow.

**Comment by Souptik-Coder** [comment]
Hi, @fluiddot I want to work on this issue. Can you assign this one to me?

**Comment by fluiddot** [comment]
> Hi, @fluiddot I want to work on this issue. Can you assign this one to me?

Hey @Souptik-Coder 👋, thank you very much for your interest in contributing to the repository. I'll assign the issue to you, please feel free to check the [_How to Contribute_ guideline](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/CONTRIBUTING.md), and please take a moment to read our [_Code of Conduct_](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/CODE-OF-CONDUCT.md). Thanks!

**Comment by harshsingh-io** [comment]
Hey @fluiddot , I think I can solve this issue and as I can see there is a long time ago issue is not resolved so can you assign me for this issue so I can work on it

**Comment by d4nnyp73** [comment]
Hey.  If nothing has happened yet, i would love to work on this issue! 

Please assign me if @harshsingh-io should not be working on it anymore.

Cheers!

**Comment by 07jasjeet** [comment]
Can I take this up?

**Comment by irfano** [comment]
> Can I take this up?

Of course! @Souptik-Coder and @harshsingh-io, I guess you're not currently working on this issue. I'm assigning this to @07jasjeet.

**Comment by irfano** [comment]
👋🏻 @07jasjeet, there was a discussion that I had overlooked earlier regarding our support for posts without titles. Therefore, this issue needs further design discussion. I am sorry to tell you that I am unassigning you from the issue and removing the `good first issue` label. 
I hope you can find another issue from the [good first issue](https://github.com/wordpress-mobile/WordPress-Android/labels/good%20first%20issue) list.

**Comment by 07jasjeet** [comment]
> 👋🏻 @07jasjeet, there was a discussion regarding our support for posts without titles. Therefore, this issue needs further design discussion. I am sorry to tell you that I am unassigning you from the issue and removing the `good first issue` label. 
> 
> I hope you can find another issue from the [good first issue](https://github.com/wordpress-mobile/WordPress-Android/labels/good%20first%20issue) list.

No problem, thanks for providing the info beforehand.
